public/templates/engines_tab.html

```diff
@@ -0,0 +1,31 @@
+<div class="engines tab">
+    <h3>select search engines</h3>
+    <p class="description">
+        Select the search engines from the list of engines that you want results
+        from
```

Comment:
```suggestion
        Select the search engines from the list of engines that you want results from.
```

Reply:
Thank you for making this change is that this will make the code a little less readable because the sentence will be very long. Anyone working on this file will have to scroll horizontally to see the whole sentence, which is not good, What do you say?